### ISSUE TITLE
memcache protocol docs incomplete for get() method

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3688]
|<img alt="rbd's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| rbd reported|
|-|-|
|Trac ID|trac#3688|
|Type|defect|
|Created|2009-03-12 18:07:00Z|
|Branch|https://github.com/twisted/twisted/tree/memcache-get-docs-3688|

The memcache protocol docs for the get() method don't mention what is returned if the key input doesn't exist.

Docs at: http://twistedmatrix.com/documents/current/api/twisted.protocols.memcache.MemCacheProtocol.html#get

They should state something like:

If the key is not found, (0, None) will be returned if withIdentifier is set to False, and (0, None, None) will be returned if withIdentifier is set to True.

(note I'm not sure about that last return value there)

<details><summary>Searchable metadata</summary>

```
trac-id__3688 3688
type__defect defect
reporter__rbd rbd
priority__normal normal
milestone__ 
branch__branches_memcache_get_docs_3688 branches/memcache-get-docs-3688
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__memcached__documentation__get memcached, documentation, get
time__1236881220000000 1236881220000000
changetime__1237572286000000 1237572286000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [26373]) Branching to 'memcache-get-docs-3688'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [26374]) Add docs for missing value case

refs [#3688](https://github.com/twisted/twisted/issues/3688)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun removed owner|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/builders/documentation/builds/331)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Looks good to me, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [26482]) Merge memcache-get-docs-3688

Author: exarkun
Reviewer: thijs
Fixes: [#3688](https://github.com/twisted/twisted/issues/3688)

Add API documentation for the return value of the memcache client's `get` method
in the case where the requested key is not present on the server.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
